### PR TITLE
LPS-130988 Rename processStatuses query param to statuses

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**

### CODE REVIEW

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:stop

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/rest-openapi.yaml [hunk 1]
```diff
@@ -940,7 +940,7 @@ paths:
                       format: date-time
                       type: string
                 - in: query
-                  name: processStatuses
+                  name: instanceStatuses
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/rest-openapi.yaml:943]: Hey @MarinhoFeliphe, The `instanceStatuses` seems a little bit redundant. Maybe we can use just `statuses`. What do you think?

Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by ramonmsp** [comment]
Just started reviewing :)

**Comment by rodrigocunhaa** [comment]
ci:reevaluate:1047401_8287

**Comment by rodrigocunhaa** [comment]
ci:forward

**Comment by rodrigocunhaa** [comment]
Just started reviewing :)

**Comment by rodrigocunhaa** [comment]
Manually forwarded here: https://github.com/brianchandotcom/liferay-portal/pull/102168

**Pull request closed without merging**